SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/Misc/SpatialActivationFlags.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "CoreMinimal.h"
+
+#include "TestDefinitions.h"
+#include "Tests/AutomationCommon.h"
+#include "Runtime/EngineSettings/Public/EngineSettings.h"
+
+static const bool s_earliestFlag =  GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
```

Comment:
Is this safe? Might the settings not be initialized at this point?

Reply:
I test this case, and apparently we get the settings we expect.
The test plugin is set to be loaded before the SpatialGDKServices.
I admit this is a bit flaky, and heavily dependent on ordering of many other parts, but this is like what the DeploymentManager is doing at the moment.